restapi/src/main/java/io/stargate/web/docsapi/service/query/filter/operation/CombinedFilterOperation.java

```diff
@@ -21,7 +21,10 @@
  *
  * @param <V> Type of a value used in the predicate.
  */
-public interface CombinedFilterOperation<V>
-    extends StringValueFilterOperation<V>,
-        DoubleValueFilterOperation<V>,
-        BooleanValueFilterOperation<V> {}
+public interface CombinedFilterOperation<V> extends BaseFilterOperation {
+  boolean test(V filterValue, Object dbValue);
```

Comment:
no let's have three test methods here as well, there is no reason to force impls to do any casts..

Reply:
Done.